README.md

```diff
@@ -173,3 +262,23 @@ Usage of /go/src/gotestwaf/gotestwaf:
       --workers int            The number of workers to scan (default 200)
       --wsURL string           WebSocket URL to check
 ```
+
+The listed options can be passed to GoTestWAF as follows:
+
+* If running the GoTestWAF Docker container, pass the configuration options in the `docker run` command after the Docker image name.
+
+    For example, to run GoTestWAF with WebSocket check, you can specify the WebSocket URL via the `wsURL` option
+    and `verbose` flag to include more information about the checking process:
+
+    ```
+    docker run -v ${PWD}/reports:/go/src/gotestwaf/reports --network="host" wallarm/gotestwaf \
+        --url=http://127.17.0.1:8080/ --wsURL=ws://127.17.0.1:8080/api/ws --verbose
```

Comment:
Different IPs: http://127.17.0.1:8080/ and ws://127.17.0.1:8080/api/ws

Should be the same, 127.0.0.1